### PR TITLE
Edraak Header and Login

### PR DESCRIPTION
**Pull request by shadinaif**
* Change header to match Edraak's look & feel
* Add edx authentication check

### CODE REVIEW

**Comment by shadinaif** [comment]
Salam @OmarEdraak , I've copied your work and apply few changes

File: dashboard/app/views/layouts/_header.html.haml [hunk 1]
```diff
@@ -137,3 +137,73 @@
...
+    var main = function() {
+      if (shouldAuthenticate()) {
+        var programsBase = location.origin.replace('://code.', '://programs.');
+        var loginUrl = programsBase + '/login/?next=' + encodeURIComponent(location.href);
+        location.href = loginUrl;
+        return  // Don't do anything else.
+      }
+
+      var userInfo = getUserInfo();
+      console.log(userInfo);
```
Comment (ghost) [inline dashboard/app/views/layouts/_header.html.haml]:
```suggestion
```

File: dashboard/app/views/layouts/_header.html.haml [hunk 2]
```diff
@@ -137,3 +137,73 @@
...
+      var headerButton = document.querySelector('#sign_in_or_user .header_button');
+      console.log(headerButton);
```
Comment (ghost) [inline dashboard/app/views/layouts/_header.html.haml]:
```suggestion
```

**Review by ghost — APPROVED** [review]

File: dashboard/app/views/layouts/_header.html.haml (hunk 1 above)
Comment (shadinaif) [reply dashboard/app/views/layouts/_header.html.haml]: Done

File: dashboard/app/views/layouts/_header.html.haml (hunk 2 above)
Comment (shadinaif) [reply dashboard/app/views/layouts/_header.html.haml]: Done

**Pull request merged**